message.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package mainflux;
+
+// RawMessage represents a message emitted by the Mainflux adapters layer.
+message RawMessage {
+	string  Channel     = 1;
+	string  Publisher   = 2;
+	string  Protocol    = 3;
+	string  ContentType = 4;
+	bytes   Payload     = 5;
+}
+
+// Message represents a resolved (normalized) raw message.
+message Message {
+	string  Channel     = 1;
+	string  Publisher   = 2;
+	string  Protocol    = 3;
+	string  Name        = 4 [json_name="n"];
```

Comment:
json_name can be removed.

Reply:
Agreed, as now all messages need to be parsed via Protobuf.